evap/student/views.py

```diff
@@ -185,7 +194,40 @@ def sorter(evaluation):
     return render(request, "student_index.html", template_data)
 
 
-def get_vote_page_form_groups(request, evaluation, preview):
+def create_voting_form(
+    request, contribution: Contribution, questionnaire: Questionnaire, preselect_no_answer: bool
+) -> QuestionnaireVotingForm:
+    initial = None
+
+    if preselect_no_answer:
+        initial = {
+            answer_field_id(contribution, questionnaire, question): NO_ANSWER
+            for question in questionnaire.rating_questions
+        }
+
+    return QuestionnaireVotingForm(
+        request.POST or None, contribution=contribution, questionnaire=questionnaire, initial=initial
+    )
+
+
+def create_voting_forms(
+    request, contribution: Contribution, questionnaires: QuerySet[Questionnaire], preselect_no_answer: bool
+) -> list[QuestionnaireVotingForm]:
+    return [
+        create_voting_form(
+            request,
+            contribution,
+            questionnaire,
+            preselect_no_answer=(preselect_no_answer and not questionnaire.is_dropout),
+            # dropout questionnaires should not be preselected
+        )
+        for questionnaire in questionnaires
+    ]
+
+
+def get_vote_page_form_groups(
+    request, evaluation: Evaluation, preview: bool, preselect_no_answer: bool
```

Comment:
same here with keyword only: someone could pass a bool for dropout when they wanted to pass one for preview